docs/guide/protocols/plainmd.rst

```diff
@@ -1,4 +1,6 @@
 Plain MD Protocol
 =================
 
-issue #743
\ No newline at end of file
+The plain MD protocol enables the user to run an MD simulation of a `ChemicalSystem`, which can contain e.g. a solvated protein-ligand complex, ora ligand and water. 
```

Comment:
Beyond my previous comment:

1. we should make sure that our first in-text reference to the Protocol points to the Protocol API (i.e. :class:`.PlainMDProtocol`.
2. Here we should talk about vacuum simulations too (since they are possible), this probably would fall under "simulation overview" or "scientific details" - something along the lines of "if you have a SolventComponent you will be running under these conditions and these settings are relevant. If you don't then you will be running in vacuum and you are expected to set this setting".